src/include/miopen/graphapi/execution_plan.hpp

```diff
@@ -62,10 +61,17 @@ class ExecutionPlan
     const EngineCfg& getEngineCfg() const noexcept { return mEngineCfg; }
     EngineCfg& getEngineCfg() noexcept { return mEngineCfg; }
     const std::vector<int64_t>& getIntermediateIds() const noexcept { return mIntermediateIds; }
-    int64_t getWorkspaceSize() const { return mWorkspaceSize; }
     std::string getJsonRepresentation() const;
 
-    void execute(const VariantPack& variantPack);
+    void execute(const VariantPack& variantPack)
```

Comment:
`miopenBackendExecute()` takes a `handle` in its arguments:
https://github.com/ROCm/MIOpen/blob/d6caccd3ec5d0ba08a3aa5f9e6df3e5780b71461/include/miopen/miopen.h#L6847

I wonder if we should add a `handle` to the signature here. I don't know exactly why CuDNN has several places to pass a `handle`. Might be because when an `execution_plan` is serialized and then restored from JSON it looses the `handle` value. For now we can have its value either from opgraph or from setAttribute call.

We can wait until we implement serialization to figure out if we need it here.